2023-11-25 01:02:13,500 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:02:16,059 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:02:39,100 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:25:12,282 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:25:16,395 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:25:46,425 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:26:22,850 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:26:23,609 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:26:51,162 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:26:51,778 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:26:57,288 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:26:58,002 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:29:05,543 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:29:06,185 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:29:14,304 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:29:46,323 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:29:51,763 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:29:52,445 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:30:00,572 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:30:02,397 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:30:07,849 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:30:08,444 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:30:08,456 - [WARNING] - app - (tasks.py).connect_to_db(12) - name 'async_sessiong' is not defined
2023-11-25 01:30:08,457 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:30:10,233 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:30:10,891 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:30:13,233 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:30:13,803 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:30:22,045 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:30:22,658 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:30:27,106 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:30:27,738 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:30:44,852 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:30:45,428 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:30:45,442 - [WARNING] - app - (tasks.py).connect_to_db(12) - 'Engine' object has no attribute '_engine'
2023-11-25 01:30:45,442 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:30:47,216 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:30:47,852 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:30:47,866 - [WARNING] - app - (tasks.py).connect_to_db(12) - 'Engine' object has no attribute '_engine'
2023-11-25 01:30:47,866 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:32:16,092 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:32:16,718 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:32:16,731 - [WARNING] - app - (tasks.py).connect_to_db(12) - 'AsyncSession' object has no attribute 'transaction'
2023-11-25 01:32:16,731 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:32:29,637 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:32:30,317 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:33:00,649 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:33:01,289 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:33:01,302 - [WARNING] - app - (tasks.py).connect_to_db(12) - 'AsyncSession' object has no attribute 'transaction'
2023-11-25 01:33:01,302 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:33:29,298 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:33:30,007 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:33:30,020 - [WARNING] - app - (tasks.py).connect_to_db(12) - 'AsyncSession' object has no attribute 'transaction'
2023-11-25 01:33:30,020 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:35:20,474 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:35:21,073 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:35:38,202 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:35:38,850 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:35:42,739 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:35:43,395 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:35:52,155 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:35:52,844 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:36:20,412 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:36:21,080 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:36:26,529 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:36:27,175 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:36:36,319 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:36:36,941 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:36:45,618 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:36:46,316 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:36:55,452 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:36:56,085 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:37:01,080 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:37:01,726 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:37:08,280 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:37:08,909 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:38:10,024 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:38:10,703 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:38:11,829 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:38:19,312 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:38:29,770 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:38:30,383 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:38:30,383 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 01:38:30,383 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:38:38,486 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:38:39,219 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:42:19,748 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:42:20,411 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:42:20,412 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 01:42:20,412 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:42:26,411 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:42:29,436 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:42:29,436 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 01:42:29,436 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:42:35,974 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:42:36,591 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:43:40,940 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:43:41,556 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:45:09,640 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:45:39,387 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:45:52,396 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:45:53,002 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:46:05,498 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:46:06,143 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:46:52,327 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:46:52,971 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:47:10,085 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:47:10,720 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:48:20,858 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:48:21,564 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:48:33,886 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:48:34,650 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:48:34,650 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 01:48:34,651 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:48:41,065 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:48:41,753 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:48:51,974 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:48:52,582 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:49:17,144 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:49:17,765 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:49:21,104 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:49:21,729 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:49:31,966 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:49:32,603 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:49:36,487 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:49:37,103 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:49:40,009 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:49:40,645 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:51:41,320 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:51:47,400 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:51:54,402 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:51:55,043 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:51:58,382 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:51:59,058 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:53:21,402 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:53:22,045 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:54:53,923 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:54:54,724 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:54:57,620 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:54:58,326 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:55:15,352 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:55:15,949 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:55:20,393 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:55:21,005 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:55:29,235 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:55:33,778 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:55:42,444 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:55:43,181 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:55:50,174 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:55:58,350 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:56:22,178 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:56:25,990 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:56:32,518 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:56:33,188 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:56:40,704 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:56:41,326 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:56:48,447 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:56:49,131 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:56:58,807 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:56:59,486 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:57:06,583 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:57:07,225 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:57:19,775 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:57:20,446 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:58:22,464 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:58:23,259 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:58:23,260 - [WARNING] - app - (tasks.py).connect_to_db(12) - name 'pool' is not defined
2023-11-25 01:58:23,260 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:58:59,396 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:59:00,072 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:59:00,084 - [WARNING] - app - (tasks.py).connect_to_db(12) - Invalid argument(s) 'max_overflow' sent to create_engine(), using configuration SQLiteDialect_aiosqlite/NullPool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
2023-11-25 01:59:00,084 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:59:10,740 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:59:11,699 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 01:59:11,718 - [WARNING] - app - (tasks.py).connect_to_db(12) - Invalid argument(s) 'max_overflow' sent to create_engine(), using configuration SQLiteDialect_aiosqlite/NullPool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
2023-11-25 01:59:11,718 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 01:59:25,823 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 01:59:26,419 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:00:11,486 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:00:12,121 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:00:17,073 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:00:17,734 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:00:17,734 - [WARNING] - app - (tasks.py).connect_to_db(12) - name 'NullPool' is not defined
2023-11-25 02:00:17,734 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:00:20,592 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:00:21,203 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:00:26,181 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:00:26,796 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:01:20,524 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:01:21,200 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:02:49,429 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:04:28,237 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:04:29,026 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:04:29,038 - [WARNING] - app - (tasks.py).connect_to_db(12) - Invalid argument(s) 'autocommit','autoflush' sent to create_engine(), using configuration SQLiteDialect_aiosqlite/NullPool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
2023-11-25 02:04:29,039 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:04:37,154 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:04:37,803 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:04:39,039 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:04:39,642 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:07:28,154 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:07:28,745 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:07:37,903 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:07:38,523 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:07:38,535 - [WARNING] - app - (tasks.py).connect_to_db(12) - Invalid argument(s) 'pool_size' sent to create_engine(), using configuration SQLiteDialect_aiosqlite/NullPool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
2023-11-25 02:07:38,535 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:07:46,639 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:07:50,481 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:07:50,493 - [WARNING] - app - (tasks.py).connect_to_db(12) - Invalid argument(s) 'pool_size','max_overflow' sent to create_engine(), using configuration SQLiteDialect_aiosqlite/NullPool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
2023-11-25 02:07:50,493 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:07:57,051 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:07:57,680 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:07:57,691 - [WARNING] - app - (tasks.py).connect_to_db(12) - Invalid argument(s) 'pool_size','max_overflow' sent to create_engine(), using configuration SQLiteDialect_aiosqlite/NullPool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
2023-11-25 02:07:57,691 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:08:12,132 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:08:12,813 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:10:02,753 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:10:03,379 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:10:16,720 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:10:17,336 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:15:50,024 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:15:50,613 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:16:15,168 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:16:15,765 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:16:15,765 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:16:15,765 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:16:34,429 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:16:35,030 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:16:35,030 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:16:35,030 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:16:38,363 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:16:38,971 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:16:38,971 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:16:38,971 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:16:59,307 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:16:59,970 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:17:18,095 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:17:18,740 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:17:23,184 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:17:23,813 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:17:33,484 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:17:34,097 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:17:34,097 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:17:34,097 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:18:06,036 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:18:06,728 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:18:06,728 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:18:06,728 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:18:12,870 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:18:13,474 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:18:13,474 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:18:13,474 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:18:41,519 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:18:42,204 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:19:09,975 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:19:10,669 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:19:10,669 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:19:10,670 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:20:17,764 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:20:19,556 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:20:19,557 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:20:19,557 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:20:29,846 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:20:30,531 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:20:30,531 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:20:30,531 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:21:02,070 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:21:02,717 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:21:02,717 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:21:02,718 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:21:22,165 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:21:22,786 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:21:22,788 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:21:22,788 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:22:02,166 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:22:02,911 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:22:02,911 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:22:02,912 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:22:23,007 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:22:23,632 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:22:23,632 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:22:23,632 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:22:36,438 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:22:37,057 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:22:37,057 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:22:37,057 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:25:58,334 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:25:58,966 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:25:58,966 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:25:58,966 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:26:00,750 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:26:01,429 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:26:01,429 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:26:01,430 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:26:07,420 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:26:08,078 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:26:08,078 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:26:08,079 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:26:28,974 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:26:29,609 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:26:29,609 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:26:29,609 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:26:44,505 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:26:45,150 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:26:45,150 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:26:45,151 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:27:25,878 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:27:26,536 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:27:26,537 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:27:26,537 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:27:35,616 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:27:36,218 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:27:36,219 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:27:36,219 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:27:43,761 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:27:44,403 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:27:44,404 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:27:44,404 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:28:07,972 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:28:08,784 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:28:08,785 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:28:08,785 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:28:49,868 - [INFO] - app - (handlers.py).stop_app(18) - Server shutting down!
2023-11-25 02:28:50,546 - [INFO] - app - (handlers.py).start_app(10) - Server started!
2023-11-25 02:28:50,547 - [WARNING] - app - (tasks.py).connect_to_db(12) - object async_generator can't be used in 'await' expression
2023-11-25 02:28:50,547 - [WARNING] - app - (tasks.py).connect_to_db(13) - --- DB CONNECTION ERROR ---
2023-11-25 02:42:05,382 - [INFO] - app - (handlers.py).start_app(8) - Server started!
2023-11-25 02:42:15,866 - [INFO] - app - (handlers.py).stop_app(15) - Server shutting down!
